src/ports/features.ts

```diff
@@ -0,0 +1,142 @@
+import { IFetchComponent } from '@well-known-components/http-server'
+import {
+  IConfigComponent,
+  ILoggerComponent,
+} from '@well-known-components/interfaces'
+
+// Note:
+// features component may be encapsulated in @well-known-components/features
+// in a future. for the time being, given it's simplicity and ease of configuration
+// it can be implemented as follow to enable customizations:
+
+export async function createFeaturesComponent(
```

Comment:
Would you mind testing the FF's component? It would also be work to be done when moving it into it's new repository.

Reply:
Sure!